src/DotVVM.Framework/Routing/RouteTableGroup.cs

```diff
@@ -1,25 +1,37 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using DotVVM.Framework.Hosting;
 
 namespace DotVVM.Framework.Routing
 {
     public class RouteTableGroup
     {
-        public Action<string, RouteBase> AddToParentRouteTable { get; private set; }
+        public Action<string, RouteBase> AddToParentRouteTable { get; }
 
-        public string GroupName { get; private set; }
-        public string RouteNamePrefix { get; private set; }
-        public string UrlPrefix { get; private set; }
-        public string VirtualPathPrefix { get; private set; }
+        public string GroupName { get; }
 
-        public RouteTableGroup(string groupName, string routeNamePrefix, string urlPrefix, string virtualPathPrefix, Action<string, RouteBase> addToParentRouteTable)
+        public string RouteNamePrefix { get; }
```

Comment:
Are you addicted to reformatting to existing code? Removing unused private setters and adding few newlines definitely won't help readability, it just creates more merge conflicts and drops history. Everyone would like these irrelevant details to be a bit different and it would not help anyone if I'd remove these newlines between properties and add one more newline before `string groupName` in my next commit.

Reply:
Maybe a bit.

It does help readability. Inconsistent formatting is distracting and the formatting in this class was inconsistent even with itself. I could either add the property and make the mess worse or reformat it. If you really want me to revert this change and stop reformatting everything I touch; fine, just say the word. This is not gonna fix itself, though.

Reply:
Yes, I want you to stop reformatting the existing code, as I have explained before (it's just harder to read the diff and makes the history less useful). I don't know what was inconsistent with this class, it was just a few properties and looked the same. Maybe they had the redundant `private set;`, but who cares about that (well, in some cases Newtonsoft.Json does, but this is not the case). And (code style) consistency is overrated, you'll never achieve it even if you would, this is not the only project you'll meet in your career. I'd better figure out how to read code that different spacing than what you'd write.

Reply:
Fair enough. I reverted the formatting changes.

If you have no other comments, please approve the PR.